generator/src/main/java/org/apache/kafka/message/ApiMessageTypeGenerator.java

```diff
@@ -355,28 +362,9 @@ private void generateHeaderVersion(String type) {
                 buffer.decrementIndent();
                 continue;
             }
-            if (type.equals("request") && apiKey == 7) {
-                buffer.printf("// Version 0 of ControlledShutdownRequest has a non-standard request header%n");
```

Comment:
We don't need the special case for ControlledShutdown v0 anymore (since it's been removed).